web/docs/database/backends.md

```diff
@@ -1,3 +1,182 @@
 ---
-title: Database Backends
+title: Databases
 ---
+
+[Entities](/docs/database/entities.md), [Operations](/docs/database/operations.md) and [Automatic CRUD](/docs/database/crud.md) together make a high-level interface for working with your app's data. Still, all that data has to live somewhere, so let's see how Wasp deals with databases.
+
+## Database configuration
+
+You can tall Wasp which database to use in the `app` declaration's `db` field:
+
+```wasp
+app MyApp {
+  title: "My app",
+  // ...
+  db: {
+    system: PostgreSQL,
+    seeds: [
+      import devSeed from "@server/dbSeeds.js"
+    ],
+    prisma: {
+      clientPreviewFeatures: ["extendedWhereUnique"]
+    }
+  }
+}
+```
+
+`app.db` is a dictionary with following fields (all fields are optional):
```

Comment:
So this piece would go into the `Options Reference` and here you could talk about the options in general or not at all, just link to the reference.

Reply:
```suggestion
`app.db` is a dictionary with the following fields (all fields are optional):
```